Projects/UOContent/Spells/Base/SpellHelper.cs

```diff
@@ -257,6 +257,17 @@ public static bool AdjustField(ref Point3D p, Map map, int height, bool mobsBloc
             return false;
         }
 
+
+        public static bool GetEastToWest(IPoint3D from,IPoint3D target)
```

Comment:
We should already have this somewhere I thought. I'll double check when I get back.

Reply:
I didnt find it anywhere. In spells there are separately the condition (and not working :) )